deploy/script/deploy.sh

```diff
@@ -96,7 +96,7 @@ for ip in ${deploy_iplist[@]};
 do
   private_key="cert/node_"${idx}".key.pri"
   cert="cert/cert_"${idx}".cert"
-  run_one_cmd "nohup ./${server_bin} server.config ${private_key} ${cert}  > ${server_bin}.log 2>&1 &" &
+  run_one_cmd "nohup ./${server_bin} server.config ${private_key} ${cert}  > ${server_bin}${idx}.log 2>&1 &" &
```

Comment:
if you remove this, once all the binaries are running at the same machine, their logs will be lost.